pkg/mdformatter/mdgen/mdgen.go

```diff
@@ -82,30 +82,12 @@ func (t *genCodeBlockTransformer) TransformCodeBlock(ctx mdformatter.SourceConte
 			if exitErr, ok := err.(*exec.ExitError); ok && exitErr.ExitCode() == expectedCode {
 				return b.Bytes(), nil
 			}
-			return nil, errors.Wrapf(err, "run %v", execCmd)
+			return nil, errors.Wrapf(err, "run %v, out: %v", execCmd, b.String())
 		}
 		return b.Bytes(), nil
 	}
 
-	lang, langOk := infoStringAttr[infoStringKeyLang]
-	typePath, typOk := infoStringAttr[infoStringKeyType]
-	if typOk || langOk {
-		if typOk != langOk {
-			return nil, errors.Errorf("got ambiguous attributes: %v. Expected is e.g ```yaml %q=<value> %q=go . Got info string %q", infoStringAttr, infoStringKeyType, infoStringKeyLang, string(infoString))
-		}
-		switch lang {
-		case "go", "golang":
-			return genGo(ctx, "", typePath)
-		default:
-			return nil, errors.Errorf("expected language a first element of info string got %q; Got info string %q", lang, string(infoString))
-		}
-	}
 	panic("should never get here")
 }
 
 func (t *genCodeBlockTransformer) Close(ctx mdformatter.SourceContext) error { return nil }
-
-func genGo(ctx context.Context, moduleRoot string, typePath string) ([]byte, error) {
```

Comment:
I thought this was to be done as mentioned in #23. Do you have something else in mind? 🙂

Reply:
I think this part is not clearly understood yet. We don't know yet how it should look.

This snippet was mainly for the beginning to show you and Uche how we might want to do it.

Normally it's not the best to maintain code which is not used (YAGNI) so killed it (:

Reply:
Oh okay!